src/MSAL.Common/TokenCacheExtensions.cs

```diff
@@ -44,27 +46,52 @@ public static void SetBeforeWrite(this TokenCache tokencache, TokenCache.TokenCa
         /// <summary>
         /// 
         /// </summary>
-        /// <param name="tokencache"></param>
+        /// <param name="tokenCache"></param>
```

Comment:
i don't know why .NET adds the param for extension methods. :( I will clean it out in the next PR where I will be removing extension methods altogether.